tools/devtool

```diff
@@ -588,6 +699,8 @@ cmd_help() {
     echo "        Prepare a new Firecracker release by updating the version number, crate "
     echo "        dependencies and credits."
     echo ""
+    help_release
+    echo ""
```

Comment:
Nit: let's add a new line before and after this for improved visibility.

Reply:
Not sure where to add those, this is what `./devtool --help` will look like
```
generate_syscall_tables <version>
        Generates the syscall tables for seccompiler, according to a given kernel version.
        Release candidate (rc) linux versions are not allowed.
        Outputs a rust file for each supported arch: src/seccompiler/src/syscall_table/{arch}.rs
        Supported architectures: x86_64 and aarch64.

    install [-p|--path] [--debug|--release]
      Install firecracker, jailer and seccomp binaries to /usr/local/bin or a given path.
      Only the musl linked binaries are supported.
        --path                Install binaries to a specified path.
        --debug               Install the debug binaries.
        --release             Install the release binaries. This is the default.

    help
        Display this help message.

    prepare_release <version>
        Prepare a new Firecracker release by updating the version number, crate 
        dependencies and credits.

    release -v <val> -u <val> [-t <val> -r <val> -m <val> -p]
        Draft a GitHub release by calling into its API.
        Firstly, a tag will be created for the release number provided, together with a
        description obtained from the CHANGELOG file. The tag will be pushed to a provided remote.
        Secondly, the script will call into GitHub's API for programatically posting a draft release.
        For authentication reasons, one needs to pass a GitHub API access token.
        You can find instructions on how to create one: https://github.blog/2013-05-16-personal-api-tokens/.
        -v, --version             The targeted release version.
        -u, --username            The GitHub owner of the targeted repo.
        -t, --token               A GitHub Access token with repo permissions
        -r, --repo                The name of the repository where the release will be posted.
        -m, --remote              The name of the remote where the release tag will be pushed.
        -p, --prerelease          Marks the release as a pre-release.

    shell [--privileged]
        Launch the development container and open an interactive BASH shell.
        -p, --privileged    Run the container as root, in privileged mode.
                            Running Firecracker via the jailer requires elevated
                            privileges, though the build phase does not.

```

You can see there are newlines before and after the "release" section I added

Reply:
Synced offline, I added a space in the code.